src/cdk/overlay/overlay.spec.ts

```diff
@@ -444,6 +444,27 @@ describe('Overlay', () => {
       expect(overlayContainerElement.querySelectorAll('.fake-positioned').length).toBe(1);
     }));
 
+    it('should have the overlay in the DOM in position strategy when reattaching', fakeAsync(() => {
+      config.positionStrategy = {
+        attach(ref: OverlayRef): void {
+          expect(ref.hostElement.parentElement)
```

Comment:
It's unlikely to happen, but this will technically pass if `attach` is never called. It might be better to put a variable on the outside and assert at the end of the test.

Reply:
Good catch!